easybuild/easyconfigs/b/BEDTools/BEDTools-2.28.0-foss-2018a.eb

```diff
@@ -0,0 +1,38 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.28.0'
+
+homepage = "https://github.com/arq5x/bedtools2"
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/releases/download/v%(version)s/']
+sources = ['bedtools-%(version)s.tar.gz']
+checksums = ['15af6d10ed28fb3113cd3edce742fd4275f224bc06ecb98d70d869940220bc32']
+
+buildopts = 'CXX="$CXX"'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
+
+files_to_copy = ["bin", "docs", "data", "genomes", "scripts", "test"]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['annotateBed', 'bamToBed', 'bamToFastq', 'bed12ToBed6', 'bedpeToBam', 'bedToBam', 'bedToIgv', 'bedtools', 'closestBed', 'clusterBed', 'complementBed', 'coverageBed', 'expandCols', 'fastaFromBed', 'flankBed', 'genomeCoverageBed', 'getOverlap', 'groupBy', 'intersectBed', 'linksBed', 'mapBed', 'maskFastaFromBed', 'mergeBed', 'multiBamCov', 'multiIntersectBed', 'nucBed', 'pairToBed', 'pairToPair', 'randomBed', 'shiftBed', 'shuffleBed', 'slopBed', 'sortBed', 'subtractBed', 'tagBam', 'unionBedGraphs', 'windowBed', 'windowMaker']],
```

Comment:
@ghuls Do you mind breaking wrapping this line to have lines that are maximally 120 characters wide?

Something like:

```python
sanity_check_paths = {
    'files': ['bin/%s' % x for x in ['annotateBed', 'bamToBed', 'bamToFastq', 'bed12ToBed6',
                                     'bedpeToBam',, 'bedToBam', 'bedToIgv', 'bedtools', 'closestBed', 
                                     'clusterBed', 'complementBed', , 'coverageBed', ...]],
    'dirs': files_to_copy,
}